<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>770</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Other</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-1</y>
        <width>771</width>
        <height>571</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab_MAME">
       <attribute name="title">
        <string>MAME</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>370</y>
          <width>160</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="CRT_Shader">
           <property name="text">
            <string>Use CRT shader</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_launch_MAME">
           <property name="text">
            <string>Launch</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>PC-88</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_11">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>320</y>
          <width>170</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <widget class="QPushButton" name="Chose_file_PC88_floppy1">
           <property name="text">
            <string>Chose Floppy 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Chose_file_PC88_floppy2">
           <property name="text">
            <string>Chose Floppy 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Chose_file_PC88_Cassete">
           <property name="text">
            <string>Chose Cassete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_PC98_2">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>PC-98</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_8">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>320</y>
          <width>170</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="Chose_file_PC98_floppy1">
           <property name="text">
            <string>Chose Floppy 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Chose_file_PC98_floppy2">
           <property name="text">
            <string>Chose Floppy 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Chose_file_PC98_CDROM">
           <property name="text">
            <string>Chose CD-ROM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_PC98">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Sony</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>771</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>PlayStation</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_3">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>380</y>
          <width>160</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="Chose_file1">
           <property name="text">
            <string>Chose Game Disk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="radioButton_NTSC_USA">
           <property name="text">
            <string>NTSC_USA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_PAL_EU">
           <property name="text">
            <string>PAL_EU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_NTSC_Japan">
           <property name="text">
            <string>NTSC_Japan</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>501</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_9">
     <attribute name="title">
      <string>Sega</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>-4</x>
        <y>-1</y>
        <width>771</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Master System</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_12">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>380</y>
          <width>170</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <widget class="QPushButton" name="Chose_file_MasterSystem">
           <property name="text">
            <string>Chose Game Cartridge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_MasterSystem">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Genesis</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_10">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>380</y>
          <width>170</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QPushButton" name="Chose_file_MegaDrive">
           <property name="text">
            <string>Chose Game Cartridge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_Megadrive">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_12">
       <attribute name="title">
        <string>Saturn</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>380</y>
          <width>160</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QPushButton" name="Chose_file_Saturn">
           <property name="text">
            <string>Chose Game Disk</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_Saturn">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>390</y>
          <width>171</width>
          <height>111</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton_NTSC_USA_Saturn">
           <property name="text">
            <string>NTSC_USA</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_PAL_EU_Saturn">
           <property name="text">
            <string>PAL_EU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_NTSC_Japan_Saturn">
           <property name="text">
            <string>NTSC_Japan</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Nintendo</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_5">
      <property name="geometry">
       <rect>
        <x>-6</x>
        <y>-1</y>
        <width>771</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_13">
       <attribute name="title">
        <string>NES</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_7">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>380</y>
          <width>170</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="Chose_file_NES">
           <property name="text">
            <string>Chose Game Cartridge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_NES">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_14">
       <attribute name="title">
        <string>SNES</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_6">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>380</y>
          <width>170</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="Chose_file_SNES">
           <property name="text">
            <string>Chose Game Cartridge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_SNES">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_15">
       <attribute name="title">
        <string>N64</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_9">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>380</y>
          <width>170</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <item>
          <widget class="QPushButton" name="Chose_file_N64">
           <property name="text">
            <string>Chose Game Cartridge</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Launcher_Button_N64">
           <property name="text">
            <string>Launch!</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_2"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_2">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>Software information.</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
